### ISSUE TITLE
Qemu DTS for VM Cross Connector?

### ISSUE THREAD

**Issue by j1mst1x** [seL4/camkes-vm-images#9]
TL;DR: What is the Qemu Virt DTS for the VM Cross Connector build?

So I am playing with a project that requires me to rebuild the rootfs with glibc instead of uClib, and I needed to extend the RAM of the guest to 4GB. Well I figured out how to extend the RAM, but I had to move the PCI regions for the cross connector. That's when I noticed the Qemu directory does not have a DTS in it with the PCI device node for the cross connector like some of the other platforms. So I got the example working and pulled it out of /proc/device-tree/pci, but I think I fat fingered something and everything is broken now :-)  I also noticed that connection.ko is from kernel 4.14.87 instead of 4.9.189. Looks like the last time someone built the example was in 2020. I was wondering is someone could push that qemu DTS for me please? (So I can confirm what I pulled out of /proc/device-tree, what I think I really want is the interrupt-map... this is my first time playing with PCI in the device tree)

I'm also in the seL4 mattermost, if anyone wants to chat beside post in issue.

**Comment by j1mst1x** [comment]
Never mind I figured it out. The FDT is being manipulated before the VM is spawned and passed in. Which is why you don't provide one. I had the 'generate_dtb' option turned off because I was troubleshooting qemu putting its PCIE bridge where I wanted the crossvm module to go.
